plugins/services/src/js/constants/ServiceErrorMessages.js

```diff
@@ -22,6 +22,12 @@ const ServiceErrorMessages = [
     type: 'SERVICE_DEPLOYING',
     message: 'The service is currently locked by one or more deployments. ' +
       'Press again to force this operation.'
+  },
+  {
+    path: /^container\.docker\.portMappings\.[0-9]+\.name$|^portDefinitions\.[0-9]+\.name$/,
```

Comment:
Let's split this into two separate messages to make the path pattern more readable.